var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -32,9 +32,9 @@ class Sqlite(Package):
        running a separate RDBMS process.
     """
     homepage = "www.sqlite.org"
+    url      = "https://www.sqlite.org/2014/sqlite-autoconf-3080500.tar.gz"
 
-    version('3.8.5', '0544ef6d7afd8ca797935ccc2685a9ed',
-            url='https://www.sqlite.org/2014/sqlite-autoconf-3080500.tar.gz')
+    version('3080500', '0544ef6d7afd8ca797935ccc2685a9ed')
```

Comment:
If that was the intent, then the developers would have named it that way. The problem with changing the version to whatever we feel like makes sense is that we now have no way of converting that to a URL. If someone adds other versions, they will have to add a URL for every version.